.github/workflows/build.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
   build:
     strategy:
       matrix:
-        runner: [windows-large, macos-12-xl]
+        runner: [windows-large, macos-12-xl, ubuntu-22.04]
```

Comment:
Like with `windows-large` we have a larger runner available for linux builds, which this build can use by selecting `linux-large` instead of the commodity `ubuntu-22.04`

![Screenshot 2024-04-06 085638](https://github.com/secondlife/viewer/assets/151138/a93accea-60ee-418b-a634-219bfa1ccfab)

Reply:
Thank you for the info. I moved that build over to the large runner. I'm not sure if that has has access to AUTOBUILD_GITHUB_TOKEN  (the previous build failed due to that).

Though it is strange anyway that ReleaseOS comes with HAVOK=On when kicked off by GHA (I verified with Windows, there ReleaseOS always builds with Havok)